ldms/src/contrib/sampler/variorum_sampler/Makefile.am

```diff
@@ -4,8 +4,8 @@ dist_man7_MANS =
 dist_man1_MANS =
 EXTRA_DIST =
 
-AM_CPPFLAGS = @OVIS_INCLUDE_ABS@ @VARIORUM_CFLAGS@ @JANSSON_CFLAGS@
-AM_LDFLAGS = @OVIS_LIB_ABS@ @VARIORUM_LDFLAGS@ @JANSSON_LDFLAGS@
+AM_CPPFLAGS = @OVIS_INCLUDE_ABS@
+AM_LDFLAGS = @OVIS_LIB_ABS@
```

Comment:
This looks good, but remember to on the LIBADD line to replace the explicit libraries with the LT* variables that AC_LIB_HAVE_LINKFLAGS creates.